etc/rc.d/network.lua

```diff
@@ -27,28 +26,32 @@ function start()
 
     component.modem.open(port)
 
-    local t = thread.create(function() 
+    local t = thread.create(function()
```

Comment:
This is a more general thing and not actually related to these changes specifically but please don't use so many abbreviations for your variables. It makes code unnecessarily harder to read.

Reply:
changed naming